python/metatensor-torch/tests/learn/tanh.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import torch
+
+import metatensor.torch
+from metatensor.torch.learn.nn import InvariantTanh, Tanh
+
+from .._tests_utils import _test_module_save_load, _test_scripted_module_output
+from ._tests_utils import random_single_block_no_components_tensor_map
```

Comment:
I don't love this =/ I would rather have some code duplication in tests but make them extremely easy to read & as simple as possible. Do you have a strong feeling here?

Reply:
I'm talking about `_test_module_save_load` and `_test_scripted_module_output`, not `random_single_block_no_components_tensor_map` (for now)

Reply:
When I was writing and editing the tests (i.e. changing the io buffer context manager) it was rather inconvenient to change all the tests and do copy-paste-editing of all the variable names etc. As such it felt appropriate to remove redundancy in this way. 

I'd argue that these two tests are simple enough that one can easily understand what they do by looking at them once in `.._tests_utils`, and so slight loss in readability makes future developments and edits easier. Also the reason they are in `.._tests_utils` is that they are the same test for all learn modules, but also in principle for all operations too, and could be applied here as well. Thoughts?